src/ProjectTemplates/README.md

```diff
@@ -62,6 +62,101 @@ Then, use one of:
 previous step, it is NOT advised that you install templates created on your local machine using just
 `dotnet new -i [nupkgPath]`.
 
+#### Conditional tests & skipping test platforms
```

Comment:
How much of this is specific to template tests? Should it be documented in another location instead?

Reply:
Yeah @HaoK said the same thing in a comment on a commit before I rebased (so it got lost from here) but I'll do that shuffle once this is in main.